docs/beyond_basics/101-147-riastores.rst

```diff
@@ -0,0 +1,704 @@
+With the sibling and special remote set up, upon an invocation of
+:command:`datalad push --to <sibling>`, the complete dataset contents, including
+annexed contents, will be published to the RIA store, with no further setup or
+configuration required [#f6]_.
+
+Advantages of RIA stores
+""""""""""""""""""""""""
+Storing datasets in RIA stores has a number of advantages that align well with
+the demands of central dataset management on shared compute infrastructure, but are also
+well suited for most back-up and storage applications.
+In a RIA store layout, the first two levels of subdirectories can host any
+number of keystores and bare repositories. As datasets are identified via ID and
+stored *next to eachother* underneath the top-level RIA store directory, the
+store is completely flexible and extendable, and regardless of the number or
+nature of datasets inside of the store, a RIA store keeps a homogeneous directory
+structure. This aids the handling of large numbers of repositories, because
+unique locations are derived from *dataset/repository properties* (their ID)
+rather than a dataset name or a location in a complex dataset hierarchy.
+Because the dataset representation in the RIA store is a bare repository,
+"house-keeping" as well as query tasks can be automated or performed by data
+management personnel with no domain-specific knowledge about dataset contents.
+Short maintenance scripts can be used to automate basically any task that is
+of interest and possible in a dataset, but across the full RIA store.
+A few examples are:
+
+- Copy or move annex objects into a 7z archive.
+- Find dataset dependencies across all stored datasets by returning the dataset
+  IDs of subdatasets recorded in each dataset.
+- Automatically return the number of commits in each repository.
+- Automatically return the author and time of the last dataset update.
+- Find all datasets associated with specific authors.
+- Clean up unnecessary files and minimize a (or all) repository with :term:`Git`\s
+  `garbage collection (gc) <https://git-scm.com/docs/git-gc>`_ command.
+
+The usecase :ref:`usecase_datastore` demonstrates the advantages of this in a
+large scientific institute with central data management.
+Due to the git-annex ora-remote special remote, datasets can be exported and
+stored as archives to save disk space.
+
+.. todo::
+
+   link to ukb chapter as example
+
+.. _riaworkflows:
+
+RIA store workflows
+^^^^^^^^^^^^^^^^^^^
+
+The user facing commands for interactions with a RIA store are barely different
+from standard DataLad workflows. The paragraphs below detail how to create and
+populate a RIA store, how to clone datasets and retrieve data from it, and also
+how to handle permissions or hide technicalities.
+
+Creating or publishing to RIA stores
+""""""""""""""""""""""""""""""""""""
+
+.. index:: ! datalad command; create-sibling-ria
+
+A dataset can be published into an existing or not yet existing RIA store by
```

Comment:
"published" is incorrect. `create-sibling-ria` doesn't publish. It only creates the sibling.

Reply:
yes, thx